tools/gce/linux_performance_worker_init.sh

```diff
@@ -135,6 +135,12 @@ curl -sSL https://get.rvm.io | bash -s stable --ruby
 source ~/.rvm/scripts/rvm
 gem install bundler
 
+# PHP dependencies
```

Comment:
the PHP dependencies. for the benchmarks to work, someone needs to manually ssh to all the workers and install this. also, you should try that the script actually successfully creates a workers.